pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/defaultcolumn/DefaultColumnHandler.java

```diff
@@ -28,4 +28,10 @@
    */
   void updateDefaultColumns()
       throws Exception;
+
+  /**
```

Comment:
(nit) The comment is a bit misleading. The DefaultColumnHandler also adds new column(s) with the default value and fwd index, dictionary if the schema has extra column(s) but segment disk metadata doesn't have them. In addition to this, yes the default value of auto-generated columns can also be updated by the handler if need be as you mentioned in javadoc